lib/DiscoveryStreamFeed.jsm

```diff
@@ -1481,7 +1489,7 @@ getHardcodedLayout = basic => {
               feed: {
                 embed_reference: null,
                 url:
-                  "https://getpocket.cdn.mozilla.net/v3/firefox/global-recs?version=3&consumer_key=$apiKey&locale_lang=en-US&feed_variant=default_spocs_on",
+                  "https://getpocket.cdn.mozilla.net/v3/firefox/global-recs?version=3&consumer_key=$apiKey&locale_lang=$locale&feed_variant=default_spocs_on",
```

Comment:
Is the server expecting `locale_lang=en-GB` as well as `locale_lang=de-CH` etc?

Reply:
Also, given the naming of `locale_lang` instead of just `locale` is the server actually expecting just "en" and "de" which are the "language" portion of a locale while the "GB" and "CH" portion is technically the "region" portion of a locale.

Reply:
This should just work, because it's using the same pattern as the old AS which wasn't a problem, but I def think it's worth double checking.

Reply:
Also, when testing I saw german feeds, but I can see a case where the locale code of de, is being interpreted as DE and en-US is being considered incorrect, and thus the server defaults to US, which would give us what we expect, but in a very incorrect way. Which if the case, has been a bug on the previous AS system too.

Reply:
Also going to see what the server does with "en-GB/en-CA/de-CH/etc"

Also question, the way you said "has only received "en-US" even for en-GB users" makes me ask the question of, are en-GB users primarily using the en-US version?

Reply:
I verified this in a test, at the very least, de-CH also pinged this question to back end, but I don't think we should wait on it.

I think I have a good idea how the back end logic works, because I tried bogus locales like:
https://getpocket.cdn.mozilla.net/v3/firefox/global-recs?version=3&consumer_key=$apiKey&locale_lang=en-AB&count=30
https://getpocket.cdn.mozilla.net/v3/firefox/global-recs?version=3&consumer_key=$apiKey&locale_lang=de-AB&count=30 

And both returned the proper language content. So pretty sure the server is looking for starts with en or de. Or maybe starts with de or default to en.